packages/roosterjs-content-model/lib/publicTypes/ContentModelEditorCore.ts

```diff
@@ -78,6 +78,7 @@ export interface ContentModelCoreApiMap extends CoreApiMap {
     setContentModel: SetContentModel;
 
     /**
+     * @deprecated
```

Comment:
Content Model interface is not stable now. So it is ok to just remove this function.